api-reference/10 UI Components/dxDiagram/1 Configuration/customShapes/textHeight.md

```diff
@@ -11,3 +11,4 @@ This property specifies a fraction from 0 to 1 that represents the height of a s
 
```

Comment:
`This property specifies a fraction from 0 to 1 that represents the height of a shape's text in relation to the shape height.`

Please rewrite without using "represents".